### PR TITLE
CVAT docs: Cvat intro video

### PR DESCRIPTION
**Pull request by arvfilippov**
added cvat intro video
updated document structure "getting started"
added new subsection "product tour"

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by zhiltsov-max** [comment]
Is it possible to replace the 7mb video file in the repo? A gif animation, a youtube link, or a slideshow would be better.

**Comment by chernov-a** [comment]
> Is it possible to replace the 7mb video file in the repo? A gif animation, a youtube link, or a slideshow would be better.

Yes, we will insert a youtube link

File: site/content/en/docs/getting_started/product_tour/cvat-intro.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: 'Cvat intro'
+linkTitle: 'Cvat intro'
+weight: 1
+---
+
+In this video, we show how to use CVAT - how to sign up, upload your data, annotate it, and download it.
+This video is the first one in a series that’ll guide you on our product and its many features.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/qyvMyDJUCmA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (nmanovic) [inline site/content/en/docs/getting_started/product_tour/cvat-intro.md:10]: Need to move the video into another youtube channel: https://www.youtube.com/channel/UCV3qghS05GjAI714ID97IuA

**Review by nmanovic — APPROVED** [review]
LGTM

**Pull request merged**